### ISSUE TITLE
 how can i get the csv result 

### ISSUE THREAD

**Issue by beijing-university** [projectdiscovery/httpx#1480]
i use the commond: httpx -u baidu.com -sc -title -csv test -csvo "utf-8"

but i get the file wrong, can u help me to get the csv file? Thanks!

**Comment by MetzinAround** [comment]
I think you still need to use the `-o` flag to name the file. 

`httpx -u baidu.com -sc -title -csv -csvo "utf-8" -o test.csv`


the -csv flag only changes the output in the terminal.

**Comment by MetzinAround** [comment]
Actually, @ehsandeep I want to double check this answer...

you have to use -o if you want to create a file that holds the output, right?

**Comment by dogancanbakir** [comment]
@MetzinAround, That's correct; `-csv` or `-json` are boolean flags to enable formatted output. `-output` is for specifying the output file.
